package.json

```diff
@@ -5,7 +5,11 @@
   "source": "src/index.ts",
   "main": "lib/preact-transitioning.js",
   "module": "lib/preact-transitioning.module.js",
-  "exports": "./lib/preact-transitioning.modern.js",
+  "exports": {
+    "types": "./lib/types/index.d.ts",
+    "require": "./lib/preact-transitioning.js",
+    "default": "./lib/preact-transitioning.modern.mjs"
```

Comment:
Even if we put `.modern.js` here, microbundle will still generate it as `.modern.mjs` instead, so that's why the extension has changed.